src/accessibility/outputs.js

```diff
@@ -422,33 +422,38 @@ function _getMiddle(f, args) {
 }
 
 //gets position of shape in the canvas
-function _getPos(args, canvasWidth, canvasHeight) {
-  if (args[0] < 0.4 * canvasWidth) {
-    if (args[1] < 0.4 * canvasHeight) {
+p5.prototype._getPos = function (x, y) {
+  const untransformedPosition = new DOMPointReadOnly(x, y);
+  const currentTransform = this.drawingContext.getTransform();
```

Comment:
I think this might not work if this gets called when in WebGL mode. But if you detect WebGL mode, you can use `new DOMMatrix(this._renderer.uMVMatrix.mat4)` and the rest should work!